packages/dashboard/src/components/eventsTable.tsx

```diff
@@ -159,6 +166,34 @@ export function EventsTable({
       field: "messageId",
       flex: 1,
     },
+    {
+      field: "relatedResources",
+      flex: 1,
+      valueGetter: (params: any) => JSON.parse(params.row.traits),
+      renderCell: ({ value }: GridRenderCellParams) => {
```

Comment:
I realize that I should have been more explicit in the issue, but also need to relate segment and journey/broadcast resources as now described in the ticket.